Libraries/LibWeb/WebAssembly/Memory.cpp

```diff
@@ -74,7 +84,8 @@ WebIDL::ExceptionOr<u32> Memory::grow(u32 delta)
     return previous_size;
 }
 
-// https://webassembly.github.io/spec/js-api/#reset-the-memory-buffer
+// https://webassembly.github.io/spec/js-api/#refresh-the-memory-buffer
+// FIXME: add memaddr argument.
```

Comment:
That "refresh the memory buffer of memaddr" AO operates one level above this Memory object (see e.g. step 3 where it gets a "memory" object - this is that object)
"Refresh the memory buffer" is written like a global AO, please either reword this fixme to require conforming to that, or split steps 1-3 to a separate static function that performs the actions.

Reply:
Oh, wow, that's quite complicated! I think I need to spend a bit more time in the codebase and the spec to understand how to implement `refresh-the-memory-buffer` as an abstract operation.

For now, I've updated the `FIXME`, hopefully it is more descriptive now.